tools/build_and_test_onnxrt.sh

```diff
@@ -26,5 +26,8 @@ pip3 install -r requirements.txt
 # Add newer cmake to the path
 export PATH="/opt/cmake/bin:$PATH"
 export CXXFLAGS="-D__HIP_PLATFORM_HCC__=1 -w"
-./build.sh --config Release --update --build --parallel --cmake_extra_defines ONNXRUNTIME_VERSION=$(cat ./VERSION_NUMBER) --test --use_migraphx
-# pip3 install /code/onnxruntime/build/Linux/Release/dist/*.whl
+./build.sh --config Release  --cmake_extra_defines CMAKE_HIP_COMPILER=/opt/rocm/llvm/bin/clang++ --update --build --parallel --cmake_extra_defines ONNXRUNTIME_VERSION=$(cat ./VERSION_NUMBER) --skip_tests --rocm_home /opt/rocm --use_migraphx --migraphx_home /opt/rocm --rocm_version=`cat /opt/rocm/.info/version-dev`
```

Comment:
This skips test.  The script use to --test.  If the fix for the failing tests is coming in a different PR, add a TODO: Turn tests back on with #xxxx

Reply:
It still builds onnxruntime_test_all

Invoking the migraphx_test_launcher.sh is what's done in the build.py script if --test is set. I confirmed this locally and verified tests are run before I pushed this.